source/administration/important-upgrade-notes.rst

```diff
@@ -7,6 +7,22 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.24.0                                            | Some Prometheus metrics being removed to be replaced by better ones                                                                                              |
```

Comment:
@agnivade I will need some help with the Prometheus upgrade note

Reply:
Here is what I'd say:

```
The `mattermost_http_request_duration_seconds` histogram metric has been removed. This information was already captured by `mattermost_api_time`, which also contains the api handler name, HTTP method, and the response code.

As an example, if you are using this

`rate(mattermost_http_request_duration_seconds_sum{server=~"$var"}[5m]) /   rate(mattermost_http_request_duration_seconds_count{server=~"$var"}[5m])` to measure average call duration, it needs to be replaced with 
`sum(rate(mattermost_api_time_sum{server=~"$var"}[5m])) by (instance) /   sum(rate(mattermost_api_time_count{server=~"$var"}[5m])) by (instance)`.
```